### PR TITLE
add entry for BACARDI

### PR DESCRIPTION
**Pull request by JuleRadtke**
new cataloque entry for BACARDI dataset

### CODE REVIEW

**Comment by d70-t** [comment]
Thanks for the PR 🎉 

What is the reason to use `thredds-x.ipsl.fr` in stead of `observations.ipsl.fr` as the host in the opendap URLs? They both refer to the same host:

```
% host observations.ipsl.fr
observations.ipsl.fr is an alias for perceval.ipsl.polytechnique.fr.
perceval.ipsl.polytechnique.fr has address 129.104.53.10
% host thredds-x.ipsl.fr
thredds-x.ipsl.fr is an alias for perceval.ipsl.polytechnique.fr.
perceval.ipsl.polytechnique.fr has address 129.104.53.10
```

If there's a reason to change to `thredds-x`, then we should probably to that for all datasets on Aeris. Otherwise, I'd recommend to use `observations` also for this PR.

**Comment by JuleRadtke** [comment]
Ah I was wondering about that, I just copied the opendap data url. But if it's better to be consistent, I change it!

**Comment by d70-t** [comment]
There's an error with the [filename casing](https://github.com/eurec4a/eurec4a-intake/pull/43/checks?check_run_id=1910965173#step:5:87).

File: HALO/main.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ sources:
 
   BACARDI:
     args:
-      path: "{{CATALOG_DIR}}/BACARDI.yaml"
+      path: "{{CATALOG_DIR}}/smart.yaml"
```
Comment (d70-t) [inline HALO/main.yaml:53]: 😬

Comment (JuleRadtke) [reply HALO/main.yaml:53]: Oh 🙈

**Review by d70-t — APPROVED** [review]

**Pull request merged**